include/defines.h

```diff
@@ -70,6 +70,27 @@
 #define PEACH  6
 #define BOWSER 7
 
+/**
+ * Options for gMenuSelection and gMenuSelectionFromQuit
+**/
+#define START_MENU_FROM_QUIT 0
+#define GAME_SELECT_MENU_FROM_QUIT 1
+#define PLAYER_SELECT_MENU_FROM_QUIT 2
+#define COURSE_SELECT_MENU_FROM_QUIT 3
+#define RACING 4
+#define OPTIONS_MENU 5
+#define ENDING_SEQUENCE 5
```

Comment:
Maybe move ENDING_SEQUENCE and CREDITS_SEQUENCE out of this list of defines into a separate list

Reply:
Whatever seems to make the most sense I'm good with.

Reply:
Think I'll leave it as is unless I can make some more sense out of the different variables using these values